libs/types/src/tabulation.ts

```diff
@@ -198,6 +198,32 @@ export interface ElectionWriteInSummary {
   contestWriteInSummaries: Record<ContestId, ContestWriteInSummary>;
 }
 
+/** ID for an unadjudicated write-in */
+export const GENERIC_WRITE_IN_ID = 'write-in';
+/** Placeholder name for an unadjudicated write-in */
+export const GENERIC_WRITE_IN_NAME = 'Write-In';
+/**
+ * Represents an unadjudicated write-in, which ultimately may not be a vote
+ * for a candidate.
+ */
+export const GENERIC_WRITE_IN_CANDIDATE: Candidate = {
+  id: GENERIC_WRITE_IN_ID,
+  name: GENERIC_WRITE_IN_NAME,
+  isWriteIn: true,
+};
+
+export const PENDING_WRITE_IN_ID = GENERIC_WRITE_IN_ID;
```

Comment:
I am leaving these two with the same IDs because they should never exist in a list together, and changing IDs risked breaking something. But I separated out `PENDING` from `GENERIC` to give them different default names, and to make code using them more readable.